compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -375,6 +375,9 @@ pub struct Body<'tcx> {
     #[type_foldable(identity)]
     #[type_visitable(ignore)]
     pub function_coverage_info: Option<Box<coverage::FunctionCoverageInfo>>,
+
+    /// Coroutine local-upvar map
+    pub local_upvar_map: IndexVec<FieldIdx, Option<Local>>,
```

Comment:
Can you say a bit more about what this map does, and in particular how it affects the operational semantics of MIR? You didn't adjust the interpreter to use this field, so -- is it correct for a backend to entirely ignore this field?